platform/avr-ravenusb/rng.c

```diff
@@ -7,6 +7,11 @@
 #include <util/delay.h>
 #include "contiki.h"
 
+#ifdef DOXYGEN_SHOULD_SKIP_THIS
+#define RNG_CONF_USE_ADC            1
+#define RNG_CONF_USE_RADIO_CLOCK    1
+#endif
```

Comment:
As far as my test goes doxygen seems to have problems with multiple defined statements (`defined(ADMUX) && defined(ADCSRA) && defined(ADCSRB) && ...` )

`contiki/platform/avr-ravenusb/rng.c:25: warning: preprocessing issue while doing constant expression evaluation: syntax error`

Looking at the code again, I noticed that both macros depend on each other. So only either `RNG_CONF_USE_ADC` or `RNG_CONF_USE_RADIO_CLOCK` should be defined.
I opted for `RNG_CONF_USE_RADIO_CLOCK` now as this is also defined in [platform/avr-ravenusb/contiki-conf.h#L94](https://github.com/contiki-os/contiki/blob/master/platform/avr-ravenusb/contiki-conf.h#L94)